packages/launch/src/hooks/use-focused-launch-modal.ts

```diff
@@ -0,0 +1,53 @@
+/**
```

Comment:
Is this hook necessary? We are just further moving and returning some methods and doing nothing extra with them? I think @razvanpapadopol also mentioned this on another PR [here](https://github.com/Automattic/wp-calypso/pull/47318#discussion_r522083683)

>There is no real benefit from extracting just a select/dispatch call if there is no extra processing that we could include in the abstraction.

Reply:
This is a good point.

I thought I'd group all of these selector/actions under the same hook, just to make the code more tidy and uniform in `packages/launch/src/launch/index.tsx`, `packages/launch/src/focused-launch/summary/index.tsx`, `packages/launch/src/focused-launch/success/index.tsx` , and `apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/attach-focused-launch.tsx`.

Happy to remove the hook and refactor its usage if you want me to proceed

Reply:
> make the code more tidy and uniform
Yeah I can see why, but I agree with Razvan on that it's not necessary and we're just further 'abstracting' code that doesn't need it which can come with it's own issues I think.

I'd say the effort vs benefit by undoing this now might not be worthwhile but definitely worth bearing in mind for future hooks.

Reply:
>undoing this now might not be worthwhile

Shouldn't be a big change, I'll see if I can fit it while working on the rest of the feedback!

Reply:
Using Launch data-store selectors directly could be also easier to maintain in the future. We can achieve some encapsulation for these once https://github.com/Automattic/wp-calypso/issues/46941 is implemented. 

However, I don't think it's a blocker for this PR to refactor.

Reply:
I went ahead and removed the hook anyway, it was a quick change to make while still being fresh about it